xml/depl_nodes.xml

```diff
@@ -6479,7 +6482,197 @@ Apply the &barcl; to a Control Node.
+     <screen>&prompt.root;salt-run --out=yaml openstack.integrate prefix=mycloud</screen>
+     <para>
+      The prefix parameter allows pools to be created with the specified
+      prefix. By using different prefix parameters, multiple cloud deployments
+      can support different users and pools on the same SES deployment.
+     </para>
+    </step>
+    <step>
+     <para>
+      A YAML file will be created with content similar to the following
+      example:
+     </para>
+     <screen>ceph_conf:
+     cluster_network: 10.84.56.0/21
+     fsid: d5d7c7cb-5858-3218-a36f-d028df7b0673
+     mon_host: 10.84.56.8, 10.84.56.9, 10.84.56.7
+     mon_initial_members: ses-osd1, ses-osd2, ses-osd3
+     public_network: 10.84.56.0/21
+cinder:
+     key: ABCDEFRaxefEMxAAW4zp2My/5HjoST2Y87654321==
+     rbd_store_pool: mycloud-cinder
+     rbd_store_user: cinder
+cinder-backup:
+     key: AQBb8hdbrY2bNRAAqJC2ZzR5Q4yrionh7V5PkQ==
+     rbd_store_pool: mycloud-backups
+     rbd_store_user: cinder-backup
+glance:
+     key: AQD9eYRachg1NxAAiT6Hw/xYDA1vwSWLItLpgA==
+     rbd_store_pool: mycloud-glance
+     rbd_store_user: glance
+nova:
+     rbd_store_pool: mycloud-nova
+radosgw_urls:
+     - http://10.84.56.7:80/swift/v1
+     - http://10.84.56.8:80/swift/v1</screen>
+    </step>
+    <step>
+     <para>
+      Upload the generated YAML file to &crow; using the UI or
+      <literal>crowbarctl</literal> CLI.
+     </para>
+    </step>
+    <step>
+     <substeps>
+      <step>
+       <para>
+        If the Salt runner is not available, you must manually create pools and
+        users to allow &cloud; services to use the SES/Ceph cluster. Pools and
+        users must be created for &o_blockstore;, &o_comp;, and
+        &o_img;. Instructions for creating and managing pools, users and
+        keyrings can be found in the <link
+        xlink:href="https://www.suse.com/documentation/suse-enterprise-storage-5/">
+        SUSE Enterprise Storage</link> Administration Guide in the Key
+        Management section.
+       </para>
+      </step>
+      <step>
+       <para>
+        After the required pools and users are set up on the &ses;/&ceph;
+        cluster, create an SES configuration file in YAML format (using the
+        example template above).
+       </para>
+       <para>
+        Upload this file to &crow; using the UI or
+        <literal>crowbarctl</literal> CLI.
+       </para>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>
+      As indicated above, the SES configuration file can be uploaded to &crow;
+      using the UI or <literal>crowbarctl</literal> CLI.
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        From the main &crow; UI, the upload page is under
+        <menuchoice><guimenu>Utilities</guimenu> <guimenu>SUSE Enterprise
+        Storage</guimenu></menuchoice>.
+       </para>
+       <para>
+        If a configuration is already stored in &crow;, it will be visible in
```

Comment:
it is visible *

Reply:
conditional tense...`will be` is correct